src/ui/FileViewer/ToggleHeader/Title/Title.jsx

```diff
@@ -1,25 +1,32 @@
 import cs from 'classnames'
+import eq from 'lodash/eq'
+import isUndefined from 'lodash/isUndefined'
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 
+import { useLocationParams } from 'services/navigation'
+import { useRepoBackfilled, useRepoFlagsSelect } from 'services/repo'
+import { useFlags } from 'shared/featureFlags'
+import Icon from 'ui/Icon'
 import MultiSelect from 'ui/MultiSelect'
-import Spinner from 'ui/Spinner'
 
 import CoverageSelect from './CoverageSelect'
 
 export default function Title({ title, children, sticky = false }) {
   return (
     <div
+      data-testid="title-wrapper-div"
       className={cs(
         { 'z-10 sticky top-[4.5rem]': sticky },
-        'flex flex-col sm:flex-row items-start sm:items-center justify-between gap-4 flex-wrap bg-white px-3 sm:px-0 py-3'
+        'flex flex-col sm:flex-row items-start sm:items-center justify-between gap-4 flex-wrap bg-white px-3 sm:px-0'
       )}
     >
       {title && (
         <span className="text-base font-semibold text-ds-gray-senary">
           {title}
         </span>
       )}
-      <div className="flex flex-row items-center justify-between gap-2">
+      <div className="flex flex-col items-center justify-between gap-2 sm:flex-row">
```

Comment:
Should this be col? 🤔

Reply:
Yes, row is applied when the view port is greater then the small breakpoint